rllib/algorithms/appo/tf/appo_tf_rl_module.py

```diff
@@ -26,6 +29,10 @@ def __init__(self, *args, **kwargs):
         self.old_pi.trainable = False
         self.old_encoder.trainable = False
 
+    @override(RLModuleWithTargetNetworks)
+    def target_networks(self):
```

Comment:
Let's rename this to `get_target_network_pairs` ?

Reply:
done